input/fsh/SD_CodedContentBundles.fsh

```diff
@@ -51,3 +51,20 @@ Description: "Demographic Coded Content Bundle (Bundle): A bundle containing ins
 * entry ^slicing.description = "Slicing based on the profile"
 * insert BundleSlice(  CodedRaceAndEthnicity,  0, 1,  CodedRaceAndEthnicity,  CodedRaceAndEthnicity,  CodedRaceAndEthnicity)
 * insert BundleSlice(  InputRaceAndEthnicity,  0, 1,  InputRaceAndEthnicity,  InputRaceAndEthnicity,  InputRaceAndEthnicity)
+
+Profile: IndustryOccupationCodedContentBundle
+Parent: Bundle
+Id: vrdr-industry-occupation-coded-bundle
+Title: "Industry Occupation Coded Content Bundle"
+Description: "Industry and Occupation Coded Content Bundle (Bundle):  A bundle containing  coded industry and occupation data for a decedent.   This bundle is information-content equivalent to the tranditional MIO files."
+* insert RequireMetaProfile(CodedContentDocument)
+* insert BundleIdentifiers
+* type 1..1
+* type only code
+* type = #collection (exactly)
+* entry.resource 1..1 // each entry must have a resource
+* entry ^slicing.discriminator.type = #profile
+* entry ^slicing.discriminator.path = "resource"
+* entry ^slicing.rules = #open
+* entry ^slicing.description = "Slicing based on the profile"
```

Comment:
This slicing pattern gets used a few times for the different bundles. Could make a ruleset similar to that in VRFM

Reply:
Done